beacon-chain/state/getters.go

```diff
@@ -63,15 +63,15 @@ func (v *ReadOnlyValidator) Slashed() bool {
 // InnerStateUnsafe returns the pointer value of the underlying
 // beacon state proto object, bypassing immutability. Use with care.
 func (b *BeaconState) InnerStateUnsafe() *pbp2p.BeaconState {
-	if b.state == nil {
-		return &pbp2p.BeaconState{}
+	if b == nil {
```

Comment:
Why'd you remove the `b.state` check?

Reply:
because if b.state is nil it will just return on the next line anyway